### PR TITLE
Added labelCallback option

### PR DESCRIPTION
**Pull request by SpazzMarticus**
I saw your comment on #39 mentioning a callback for the labels:

Reworked the labelling of levels and added `labelCallback` as an option to label levels based on other data than bandwith in kbps.
(I now also fixed a bug occuring when a `labelCallback` is set, but `labels` are not.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,9 @@ var player = new Clappr.Player({
         2: 'High', // 500kbps
         1: 'Med', // 240kbps
         0: 'Low', // 120kbps
+    },
+    labelCallback: function(playbackLevel, customLabel) {
+        return customLabel + playbackLevel.level.height+'p'; // High 720p
     }
```
Comment (leandromoreira) [inline README.md]: I'd ask you to also write this documentation at README.md (as a simple usage example)

File: src/main.js [hunk 2]
```diff
@@ -96,10 +96,31 @@ export default class LevelSelector extends UICorePlugin {
   configureLevelsLabels() {
     if (this.core.options.levelSelectorConfig === undefined) return
 
-    for (var levelId in (this.core.options.levelSelectorConfig.labels || {})) {
-      levelId = parseInt(levelId, 10)
-      var thereIsLevel = !!this.findLevelBy(levelId)
-      thereIsLevel && this.changeLevelLabelBy(levelId, this.core.options.levelSelectorConfig.labels[levelId])
+    var labelCallback = this.core.options.levelSelectorConfig.labelCallback
+    if(labelCallback && typeof labelCallback !== 'function')
```
Comment (leandromoreira) [inline src/main.js]: We usually follow the code convention of using the open brackets in the same line `if (x) {` (we need to enforce it in the future by using eslint (like Clappr did))

File: src/main.js [hunk 3]
```diff
@@ -109,14 +130,6 @@ export default class LevelSelector extends UICorePlugin {
     return foundLevel
   }
 
-  changeLevelLabelBy(id, newLabel) {
```
Comment (leandromoreira) [inline src/main.js]: Why did you remove this?

File: README.md (hunk 1 above)
Comment (SpazzMarticus) [inline README.md]: This is the README.md? Should I add more details?

**Comment by SpazzMarticus** [comment]
> I'd like to ask you to checkout the dist/ folder

What do you mean with that?

**Comment by leandromoreira** [comment]
@SpazzMarticus you committed the contents of the `dist/` folder which only do when we release a new version of this plugin. It'd be nice if you remove this `dist/` from your commits.

File: README.md (hunk 1 above)
Comment (leandromoreira) [inline README.md]: God good, I didn't notice 😜  my bad.

**Comment by leandromoreira** [comment]
Thanks @SpazzMarticus

File: src/main.js (hunk 2 above)
Comment (SpazzMarticus) [inline src/main.js]: I have now updated my repository to follow the convention, should I add another pull-request, or will the brackets be fixed in one of your commits?
